centreon/www/front_src/src/Header/Poller/PollerSubMenu/PollerSubMenu.tsx

```diff
@@ -0,0 +1,93 @@
+import { makeStyles } from "tss-react/mui";
+import { Typography, Button } from "@mui/material";
+import ExportConfiguration from "./ExportConfiguration";
+
+const useStyles = makeStyles()((theme) => ({
+  list: {
+    listStyle: "none",
+    padding: 0,
+    margin: 0,
+  },
+  listItem: {
+    listStyle: "none",
+    padding: theme.spacing(1),
+    margin: 0,
+    "&:not(:last-child)": {
+      borderBottom: `1px solid ${theme.palette.divider}`,
+    },
+  },
+  link: {
+    textDecoration: "none",
+  },
+  pollarHeaderRight: {
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "space-between",
+    [theme.breakpoints.down(768)]: {
+      flexDirection: "row",
+      gap: theme.spacing(0.5),
+    },
+  },
+  pollerDetailRow: {
+    display: "flex",
+    justifyContent: "space-between",
+  },
+  pollerDetailTitle: {
+    flexGrow: 1,
+  },
+}));
+
+export const PollerSubMenu = ({
+  closeSubMenu,
+  issues,
+  pollerCount,
+  allPollerLabel,
+  pollerConfig,
+  newExporting,
+  exportConfig,
+}) => {
+  const { classes, cx } = useStyles();
+
+  return (
+    <ul className={classes.list}>
```

Comment:
You can use the List and ListItem from MUI

Reply:
Does not fit the design

Reply:
Why ? Our design is based on Material Design

Reply:
I still have to use the classes and override basically all the styles.